Add INFO-DIR-SECTION line to texinfo document to avoid lintian error.

Index: ghdl-0.30~svn20130213/gcc/src/gcc/vhdl/ghdl.texi
===================================================================
--- ghdl-0.30~svn20130213.orig/gcc/src/gcc/vhdl/ghdl.texi	2010-01-28 07:08:48.000000000 +0100
+++ ghdl-0.30~svn20130213/gcc/src/gcc/vhdl/ghdl.texi	2013-10-27 19:43:15.748420000 +0100
@@ -4,6 +4,7 @@
 @settitle GHDL guide
 @c %**end of header
 
+@dircategory Software development
 @direntry
 * ghdl: (ghdl).         VHDL compiler.
 @end direntry
